6.5_group_by.py

```diff
@@ -0,0 +1,20 @@
+def group_by(f, iterable):
+    """
+    get a function and an iterable and returns a dictionary whose keys are
+     the activation of the function on each member and the values are the members
+    :param f: function for key
+    :param iterable: iterable
+    :return: group dictionary by function
+    """
+    group_dictionary = {}
+    for i in iterable:
```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.

Reply:
I didn't change it.
Since the key in the dictionary is a number and the value is a list, there is a difference if I insert the first element into the list or if an element already exists.
And although it is possible to write dictionary comprehension with if and else, I still haven't been able to write a dictionary comprehension expression that will work properly and look nice.
Therefore, in such a case, I think that a phrase that is not too long and more readable would be better